.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,16 @@
+name: Gradle wrapper validation
+
+on:
+  push:
+  pull_request:
```

Comment:
Should we keep the `paths` tho? Is there You don't need to run if the wrapper hasn't changed.

Reply:
yeah, the OSSF "binary artifacts" check only goes green if it is run on every PR that is merged to main (it looks at CI runs for PRs that are merged to main and verifies that gradle wrapper validation was performed)

Reply:
There might also be some trickery (that I'm not specifically aware of) that could convince a gradle wrapper to run from a different location, so this is probably still nice to have.